### PR TITLE
Pytree for unix

### PR DESCRIPTION
**Pull request by monodo**
adds binaries for Debian 9 

@kalbermattenm please review & merge. Your py3 branch is already merged in this one

### CODE REVIEW

File: example_config.yaml [hunk 1]
```diff
@@ -1,15 +1,15 @@
 vars:
     debug: true
     server_url: http://localhost:5000
-    log_folder: <path_to>/pytree/log
+    log_folder: /var/www/vhosts/yverdon-geoportail/private/pytree/log
```
Comment (kalbermattenm) [inline example_config.yaml]: Sure about that ?

File: example_config.yaml [hunk 2]
```diff
@@ -1,15 +1,15 @@
...
     gmf_url: ""
     # path to cpotree exectuable
-    cpotree_executable: <path_to>/PotreeElevationProfile.exe
+    cpotree_executable: /var/www/vhosts/yverdon-geoportail/private/pytree/cPotreeDebian.out
```
Comment (kalbermattenm) [inline example_config.yaml]: Sure about that ?

File: example_config.yaml [hunk 3]
```diff
@@ -1,15 +1,15 @@
...
     # Available point clouds. NOTE: pointclouds CRS must be the same as the APP, reprojection is not implemented!
     pointclouds:
-        sitn2016: <path_to>/cloud.js
+        sit: /var/www/vhosts/yverdon-geoportail/htdocs/lidar/lidar_app/pointclouds/web/cloud.js
```
Comment (kalbermattenm) [inline example_config.yaml]: Sure about that ?

File: pytree/templates/home.html [hunk 4]
```diff
@@ -12,7 +12,7 @@ <h1>pytree server</h1>
         Well done, it looks like your wsgi flask pytree server is installed correctly.
         </p>
         <p>
-        <a href="{{ url_for('get_profile') }}?minLOD=0&maxLOD=5&width=10&coordinates={2551900,%201203475},{2555100,%201206475}&pointCloud=sitn2016&attributes=%27" target=_blank>This example</a> request should return a json header followed by a (binary) array buffer to use in GeoMapFish 2.3 using following GET parameters:
+        <a href="{{ url_for('get_profile') }}?minLOD=0&maxLOD=5&width=10&coordinates={2538598,%1180300},{2539573,%1181290}&pointCloud=sit&attributes=%27" target=_blank>This example</a> request should return a json header followed by a (binary) array buffer to use in GeoMapFish 2.3 using following GET parameters:
```
Comment (kalbermattenm) [inline pytree/templates/home.html:15]: Please revert this

File: pytree/views.py [hunk 5]
```diff
@@ -46,6 +46,7 @@ def get_profile():
     point_cloud = request.args['pointCloud']
     file = point_clouds[point_cloud]
     attributes = [request.args['attributes']]
+    print('[' + cpotree + '-' + file + '--stdout]' + str(attributes) + '["--coordinates",' +  str(polyline) + ', "--width",' +  width + ', "--min-level",' +  minLevel + ', "--max-level",' +  maxLevel + ']')
```
Comment (kalbermattenm) [inline pytree/views.py]: Sorry, no `print` here.

**Review by kalbermattenm — CHANGES_REQUESTED** [review]

**Comment by kalbermattenm** [comment]
You used my Python 3 branch for this, right ?

Because all my commits are in your as well and I first would like to merge #12...

So, as I understand the whole thing, you are definitively also using Python 3 (and I already made a Python 2.7 legacy release before merging #12 & this PR) ?

File: pytree/templates/home.html (hunk 4 above)
Comment (monodo) [reply pytree/templates/home.html:15]: https://www.youtube.com/watch?v=_Qg3Rk-B09o

**Comment by monodo** [comment]
Yep, no more python 2.x
Please merge your branch first  and I'll update the PR after

**Comment by kalbermattenm** [comment]
Could you rebase your branch on the current master (#12 was merged), because I am a bit lost with all these changes. Moreover, did you add anything to the README file concerning Linux ?

**Comment by monodo** [comment]
Done

**Pull request closed without merging**